cleanlab/count.py

```diff
@@ -105,13 +110,22 @@ class 0, 1, ..., K-1. `pred_probs` should have been computed using 3 (or
 
        TL;DR: use this method to get the most accurate estimate of number of label issues when you don't need the indices of the label issues.
 
+    multi_label : bool, optional
+       Refer to documentation for this argument in ``count.compute_confident_joint()`` with `multi_label=True` for details.
```

Comment:
need to link to function properly

Reply:
Fixed: https://github.com/aditya1503/cleanlab/blob/a684583aa0e89de6414172c891b1db305e31ade3/cleanlab/count.py#L114